apps/api-gateway/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,55 @@
+import { Controller, Get, Post, Req, Res, UseGuards } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { Request, Response } from 'express';
+import { AuthService } from './auth.service';
+import { GithubGuard } from './guards/github.guard';
+import { JwtRefreshGuard } from './guards/jwt-refresh-auth.guard';
+import { Payload } from './types/jwt.type';
+import { GithubUser } from './types/user.type';
+
+@Controller('auth')
+export class AuthController {
+  public constructor(
+    private readonly authService: AuthService,
+    private readonly configService: ConfigService,
+  ) {}
+
+  @Get('github')
+  @UseGuards(GithubGuard)
+  private async gituhblogin() {}
```

Comment:
오타 + 카멜케이스로 표현해줘야겠네요~
```suggestion
  private async githubLogin() {}
```

Reply:
[a89dd2f](https://github.com/GrebbMe/grabbme-be/pull/53/commits/a89dd2ffafe97b6830ae92a6a22df960825b56d7)
헤딩 커밋에 오타 수정 반영하였습니다

Reply:
오타는 수정이 안됐네요~

Reply:
[0b34eec](https://github.com/GrebbMe/grabbme-be/pull/53/commits/0b34eec82d49cac2452bbaecf1f0bef57b752524)
제가 까막눈인거 같습니다...
위 커밋에서 수정하였습니다